apps/allocations/app/store/init.js

```diff
@@ -0,0 +1,35 @@
+import addressBookAbi from '../../../shared/json-abis/address-book'
+import { app, handleEvent } from './'
+
+export const initStore = addressBookAddress => {
+  const addressBookApp = app.external(addressBookAddress, addressBookAbi)
+
+  const initialState = {
+    accounts: [],
+    entries: [],
+    addressBook: addressBookAddress,
+  }
+
+  return app.store(
+    async (state, event) => {
+      // ensure there are initial placeholder values
+      if (!state) state = initialState
+
+      try {
+        const next = await handleEvent(state, event)
+        const nextState = { ...initialState, ...next }
+        // Debug point
+        console.log('[Allocations store]', nextState)
+        return nextState
+      } catch (err) {
+        console.error('[Allocations script] initStore', event, err)
```

Comment:
I'm getting an error thrown by this line. when I execute the `runpayout` transaction and the account balance doesn't update. However, when I clean the cache in the aragon wrapper, The event passes through successfully and the allocations account balance updates to the correct value. The error is as follows:

`[Allocations script] initStore {logIndex: 0, transactionIndex: 0, transactionHash: "0x7e26226963c93531592a4baf1656c16abb379eabf2382f5df11220787c90efbf", blockHash: "0xec23d32ddd50989c86759d2e08675ece620ced40fb3b452ab072933c0eac70d5", blockNumber: 73, …} TypeError: argument is not a function. Are you looking for mapTo()?`